README.md

```diff
@@ -20,18 +20,20 @@ If you don't have a Snowflake account yet, you can [sign up for a 30-day free tr
 You can use [miniconda][miniconda], [anaconda][anaconda], or [virtualenv][virtualenv]
 to create a Python 3.8, 3.9, 3.10 or 3.11 virtual environment.
 
+For Snowpark pandas, only Python 3.9, 3.10, or 3.11 is supported.
```

Comment:
Why is 3.8 not supported?

Reply:
Snowpark pandas is based on pandas version 2.2.1, which in and of itself does not support Python 3.8

Reply:
Do you think it's worth adding that to the README?

Reply:
I don't think it's required. I found it a little surprising since python 3.8 is still supported until October, but looking in to it more it looks like pandas and numpy use a difference [support cadence](https://numpy.org/neps/nep-0029-deprecation_policy.html#support-table).